requirements-dev.txt

```diff
@@ -0,0 +1,4 @@
+black
+flake8
+pre-commit
```

Comment:
```suggestion
wheel
pre-commit
```

1. devs might need to build wheels of PyFstat itself anyway
1. It seems like `pre-commit` pulls in `pyyaml` which then, at least in the virgin venv I tried this out in, can't be built without having the `wheels` package at hand:
```
Building wheels for collected packages: black, pyyaml
  Running setup.py bdist_wheel for black ... done
  Stored in directory: /home/dkeitel/.cache/pip/wheels/6e/10/b5/edf7359c2edd0305cce7e3f96e07daf7ce55dceac9d3ce3373
  Running setup.py bdist_wheel for pyyaml ... error
  Complete output from command /home/dkeitel/git/sources/PyFstat/devtest/bin/python3 -u -c "import setuptools, tokenize;__file__='/tmp/pip-install-duh50s2i/pyyaml/setup.py';f=getattr(tokenize, 'open', open)(__file__);code=f.read().replace('\r\n', '\n');f.close();exec(compile(code, __file__, 'exec'))" bdist_wheel -d /tmp/pip-wheel-ndjjohth --python-tag cp37:
  usage: -c [global_opts] cmd1 [cmd1_opts] [cmd2 [cmd2_opts] ...]
     or: -c --help [cmd1 cmd2 ...]
     or: -c --help-commands
     or: -c cmd --help
  
  error: invalid command 'bdist_wheel'
  
  ----------------------------------------
  Failed building wheel for pyyaml
  Running setup.py clean for pyyaml
Successfully built black
Failed to build pyyaml
```

Reply:
> Testing the packaging/distribution and a test.pypi.org upload on this branch, the `bin` directory is not included automatically and so there are no problems. But if in the future we want to include actual executables in releases, that will then require some exclusion rules. If you know a good standard directory name for "executables that are useful for devs but not to be distributed", it would be ideal to rename it now; otherwise, we'll leave it for future us to figure out.

I'm not aware of a standard name for that. We can leave it as `bin` and use `pyfstat_cli` for those "future executables" as done by [bilby](https://git.ligo.org/lscsoft/bilby/-/tree/master/cli_bilby) (this way we also avoid to use the word `dev`, which stands for `device` as well).

Reply:
Ok let's leave it at that.